gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=1.1.1
+version=1.2.0
```

Comment:
Do we need to bump to next minor version?

Reply:
Let's release with next major version. Changed with [e2c8cc2](https://github.com/ballerina-platform/module-ballerina-edi/pull/29/commits/e2c8cc2b12ec6c15689215449f1582162e90a5de).